packages/prover/src/utils/evm.ts

```diff
@@ -181,7 +181,7 @@ export async function executeVMCall({
     gasLimit: hexToBigInt(gas ?? block.gasLimit),
     gasPrice: hexToBigInt(gasPrice ?? maxPriorityFeePerGas ?? "0x0"),
     value: hexToBigInt(value ?? "0x0"),
-    data: data ? hexToBuffer(data) : undefined,
+    data: input ? hexToBuffer(input) : data ? hexToBuffer(data) : undefined,
```

Comment:
For future reference, `data` is deprecated and `input` is the new data field for ethereum transactions. 

https://github.com/ethereum/go-ethereum/blob/0004c6b229b787281760b14fb9460ffd9c2496f1/internal/ethapi/transaction_args.go#L47-L51